testing/ef_tests/src/handler.rs

```diff
@@ -169,8 +256,16 @@ impl<E: EthSpec + TypeName> Handler for SanityBlocksHandler<E> {
     fn handler_name() -> String {
         "blocks".into()
     }
+
+    fn is_enabled_for_fork(&self, _fork_name: ForkName) -> bool {
+        // FIXME(altair): v1.1.0-alpha.3 doesn't mark the historical blocks test as
+        // requiring real crypto, so only run these tests with real crypto for now.
```

Comment:
```suggestion
        // requiring real crypto, so only run these tests with fake crypto for now.
```

Reply:
The comment is right, these tests can only be run with real crypto because they do real signature aggregation to compute sync committees. The `cfg` is `not(feature = "fake-crypto")`, i.e. real.

https://github.com/sigp/lighthouse/blob/967d680a9801bcb83676f8cdb6825b23580148cc/testing/ef_tests/src/handler.rs#L261-L263

Reply:
Ohh I just saw the ambiguous interpretation along the lines of

"we don't _have_ to use real crypto, so use fake"

when what was meant was:

"we need to use real crypto but the tests don't state that, so temporarily hardcode it"

Reply:
I reckon we leave it as-is for now and see how things change in alpha.4 (I'll open an issue on the spec if it's not fixed)